File: devcamp2018/app/src/main/java/com/psato/devfest/presentation/bindadapter/ViewBinderAdapter.java
package com.psato.devfest.presentation.bindadapter;

import android.databinding.BindingAdapter;
import android.view.View;

public class ViewBinderAdapter {

    @BindingAdapter("android:visibility")
    public static void setVisibility(View view, Boolean visibility){
        if(visibility){
            view.setVisibility(View.VISIBLE);
        }else{
            view.setVisibility(View.GONE);
        }
    }
}

File: devcamp2018/app/src/main/java/com/psato/devcamp/presentation/bindadapter/ViewBinderAdapter.kt
package com.psato.devcamp.presentation.bindadapter

import android.databinding.BindingAdapter
import android.view.View

@BindingAdapter("android:visibility")
fun setVisibility(view: View, visibility: Boolean) {
    if (visibility) {
        view.visibility = View.VISIBLE
    } else {
        view.visibility = View.GONE
    }
}

